src/Search/Search.stories.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import { Search } from '../index.js'
+import { Story } from '@storybook/react'
+import styled from 'styled-components'
+import { action } from '@storybook/addon-actions'
+
+export default {
+  title: 'Components/Search',
+  component: Search,
+}
+
+const DefaultStoryTemplate: Story = (args) => {
+  return (
+    <SearchHost
+      background={args.color}
+      containerWidth={args.containerWidth}
+      fontSize={args.fontSize}
+    >
+      <Search
+        placeholder={args.placeholder}
+        color={args.color}
+        background={args.background}
+        onSubmit={action('submit')}
+      />
+    </SearchHost>
+  )
+}
+export const Default = DefaultStoryTemplate.bind({})
```

Comment:
i'm not using template so not sure maybe i should create a story to update the one that don\t and from now use it? 

```js
export const Default: Story<typeof AppBar> = (props) => (
  <AppBar {...props}>
    <AppBar.Item active={true}>item one</AppBar.Item>
    <AppBar.Item>item two</AppBar.Item>
  </AppBar>
)
```

Reply:
I just took that from the stroybook docs 🤷‍♂️